docs/.vuepress/config.ts

```diff
@@ -1,7 +1,7 @@
 import { defineUserConfig } from "vuepress";
 import docsearchPlugin from "@vuepress/plugin-docsearch";
 import theme from "./theme.js";
-import { pwaPlugin } from '@vuepress/plugin-pwa'
+//import { pwaPlugin } from '@vuepress/plugin-pwa'
```

Comment:
Why?

Reply:
Was getting an error, running it.

Reply:
Which error exactly? The PWA only works on production die to start_url on manifest. But is there ana actual error breaking the app, or just console errors?

Also, don't simply remove stuff like this - always find out who added and ask about it.